### PR TITLE
[web] [tests] Changing configurations for firefox

### PR DESCRIPTION
**Pull request by nturgut**
The configurations for firefox is added. Previously only chrome related conf. was loaded this was causing firefox to break on "felt test browser=firefox"

New prefs are added to firefox:

- dom.disable_beforeunload=true: firefox does not show a popup for default browser
- toolkit.startup.max_resumed_crashes=999999:  firefox does not show a popup after browser crash

related to issue: https://github.com/flutter/flutter/issues/44959

### CODE REVIEW

File: lib/web_ui/dart_test_firefox.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Sources of inspiration for this file:
```
Comment (yjbanov) [inline lib/web_ui/dart_test_firefox.yaml]: I'm not sure what "inspiration" means. Does it mean you invented a bespoke new configuration format that's based on those other files? Or do you mean that this file is used by the test runner and its contents are borrowed from those other files?

File: lib/web_ui/dev/test_runner.dart [hunk 2]
```diff
@@ -83,6 +83,8 @@ class TestCommand extends Command<bool> {
 
   String get browser => argResults['browser'];
 
+  bool get isChrome => argResults['browser'] == 'chrome';
```
Comment (mdebbar) [inline lib/web_ui/dev/test_runner.dart]: This isn't being used anywhere

File: lib/web_ui/dart_test_firefox.yaml (hunk 1 above)
Comment (nturgut) [reply lib/web_ui/dart_test_firefox.yaml]: done. changed the comments on both files!

File: lib/web_ui/dev/test_runner.dart (hunk 2 above)
Comment (nturgut) [reply lib/web_ui/dev/test_runner.dart]: done. Thanks!

**Pull request merged**